Automatic Validation ended with:
> 2025-01-16T20:47:52.4540658Z ##[error] Manifest is invalid. (Line: 23, Col: 17, Idx: 1074) - (Line: 23, Col: 17, Idx: 1074): While scanning a plain scalar value, found invalid mapping.
 2025-01-16T20:47:52.4553534Z ##[error] Manifest is invalid. (Line: 23, Col: 17, Idx: 1074) - (Line: 23, Col: 17, Idx: 1074): While scanning a plain scalar value, found invalid mapping.

(Automated response - build 943.)

--- a/manifests/j/JeniusApps/Ambie/4.7.8.0/JeniusApps.Ambie.installer.yaml
+++ b/manifests/j/JeniusApps/Ambie/4.7.8.0/JeniusApps.Ambie.installer.yaml
@@ -20,7 +20,7 @@ Installers:
   InstallerUrl: https://github.com/jenius-apps/ambie/releases/download/v4.7.8/Ambie_4.7.8.0_x64_ARM64.msixbundle
   InstallerSha256: C5ED060FA98274C0E549536EC037FCFB187BA6F7EF40DF5C9F6CBD00C18E889E
   SignatureSha256: 638B79B1FFC2D7165FA0A3ABB6FE2FD9E02C992DCC3C6D2187D8330D600258B3
-    Dependencies:
+  Dependencies:
     PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.arm64
 ManifestType: installer

